ethereum_history_api/oracles/README.md

```diff
@@ -16,6 +17,8 @@ fn get_header_oracle(chain_id: Field, block_no: Field) -> (BlockHeaderPartial, B
 fn get_account_oracle(chain_id: Field, block_no: Field, address: Address) -> AccountWithStateProof;
 fn get_proof_oracle(chain_id: Field, block_no: Field, address: Address, storage_key: Bytes32) -> StateAndStorageProof;
 fn get_receipt_oracle(chain_id: Field, block_no: Field, tx_idx: Field) -> (TxType, TxReceipt, TxReceiptProof);
+fn get_transaction_oracle(chain_id: Field, block_number: u64, tx_idx: Field) -> (TxType, Transaction, TransactionProof);
```

Comment:
is there any reason why other oracle functions are taking Field and this u64?

Reply:
It is because we only recently decided to have block numbers as u64, so it wasn't yet changed in its earlier occurrences.